hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoTransferHandler.java

```diff
@@ -62,7 +65,7 @@ public CryptoTransferHandler() {
      */
     public void validate(@NonNull final TransactionBody txn) throws PreCheckException {
         // FUTURE: Migrate validation from CryptoTransferTransistionLogic.validateSemantics()
-        //        throw new UnsupportedOperationException("Not implemented");
+        throw new UnsupportedOperationException("Not implemented for transaction {}" + txn);
```

Comment:
I assume you can remove the `{}`

Reply:
maybe better % and put a String.format()

Reply:
or use .formatted, which reads better.

Reply:
Removed txn from message